less/mixins.less

```diff
@@ -1 +1,38 @@
 // Mixins below variables
+.flexbox-row(@justify:center, @align:center){
+    display: flex;
+    justify-content: @justify;
+    align-items: @align;
+}
+.flexbox-column(@justify:center, @align:center){
```

Comment:
is there a way to combine `.flexbox-row` and `.flexbox-column` into only one mixin?